### PR TITLE
Feature #530 - add Terms and Conditions page

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will add Terms and Conditions page to wallet "Hi" Menu. It will complete "Terms and Conditions" part of issue #531 

terms

### CODE REVIEW

File: packages/bierzo-wallet/src/routes/registerName/index.stories.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Layout from "./components/index";
 
 export const REGISTER_USERNAME_STORY_PATH = `${WALLET_ROOT}/Register Username`;
 export const REGISTER_USERNAME_REGISTRATION_STORY_PATH = "Register Username";
-const EGISTER_USERNAME_CONFIRMATION_STORY_PATH = "Registeration confirmation";
+const REGISTER_USERNAME_CONFIRMATION_STORY_PATH = "Registeration confirmation";
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/routes/registerName/index.stories.tsx]: Typo in `Registeration`

File: packages/bierzo-wallet/src/routes/terms/components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+const addLinkTo = (link: string): JSX.Element => <Link to={link}>{link}</Link>;
+
+export default (): JSX.Element => {
+  return (
+    <React.Fragment>
+      <Title>Terms</Title>
+      <Section>
+        <Paragraph>
+          The website {addLinkTo("https://wallet.hugnet.iov.one")} (hereinafter: the “Website”) is owned and
```
Comment (webmaster128) [inline packages/bierzo-wallet/src/routes/terms/components/index.tsx]: Can we pull out the website name into the configuration? I think we don't need the link here (user is already on that page). Just a "website name"

Comment (albertandrejev) [reply packages/bierzo-wallet/src/routes/terms/components/index.tsx]: So what should I put here? Just remove the link?

Comment (webmaster128) [reply packages/bierzo-wallet/src/routes/terms/components/index.tsx]: Jap, `wallet.hugnet.iov.one` is fine for now as long as it is easy to configure

**Comment by webmaster128** [comment]
@apanizo can you also have a look at the react parts?

File: packages/bierzo-wallet/src/routes/terms/components/index.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+    getWebsiteName();
+  }, []);
+
+  return (
+    <React.Fragment>
+      <Title>Terms</Title>
+      <Section>
+        <Paragraph>
+          The website {websiteName} (hereinafter: the “Website”) is owned and operated by the company IOV SAS,
```
Comment (albertandrejev) [inline packages/bierzo-wallet/src/routes/terms/components/index.tsx]: Hope I understand you properly.

Comment (webmaster128) [reply packages/bierzo-wallet/src/routes/terms/components/index.tsx]: Nice, thanks

**Pull request merged**